### PR TITLE
Enable the binding usage controller metrics in Kyma

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Enable the binding usage controller metrics in Kyma

After merging this PR we will have dedicated dashboard for Service Binding Usage Controller components 

Screen Shot 2019-05-09 at 17 12 22

**Related issue(s)**

- #3487

### CODE REVIEW

File: resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  ports:
+  - name: metrics
```
Comment (aszecowka) [inline resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml]: https://istio.io/docs/setup/kubernetes/prepare/requirements/
shouldn't port name start with http?

File: resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    port: 8080
```
Comment (aszecowka) [inline resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml]: shouldn't we use {{ .Values.service.internalPort }} ?

File: resources/service-catalog-addons/charts/service-binding-usage-controller/templates/deployment.yaml [hunk 3]
```diff
@@ -3,12 +3,16 @@ kind: Deployment
 metadata:
   name: {{ template "fullname" . }}
   labels:
-    app: {{ template "name" . }}
+    name: {{ template "fullname" . }}
```
[the remark this answers is not in the document]
Comment (mszostok) [reply resources/service-catalog-addons/charts/service-binding-usage-controller/templates/deployment.yaml]: I will revert that, it was introduced definitely by mistake

File: resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml (hunk 1 above)
Comment (mszostok) [reply resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml]: It works, but to satisfy the istio I will rename that to `http-metrics`

File: resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml (hunk 2 above)
Comment (mszostok) [reply resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml]: agree, fixed

**Pull request merged**